src/pages/taskQueue/TaskQueueTable.tsx

```diff
@@ -79,9 +105,24 @@ export const TaskQueueTable = () => {
     <Table
       columns={columns}
       rowKey={({ id }: { id: string }): string => id}
+      rowSelection={{
+        hideSelectAll: true,
+        selectedRowKeys: [taskId],
+        renderCell(...[, { id }]) {
+          if (id === taskId) {
+            return <div ref={taskRowRef} />;
+          }
+          return null;
+        },
```

Comment:
you can also do this in a 1 liner. 
` return id === taskId ? <div ref={taskRowRef} /> : null;`